src/spectrecoff-cli/commands/Status.fs

```diff
@@ -0,0 +1,46 @@
+namespace SpectreCoff.Cli.Commands
+
+open System.Threading.Tasks
+open Spectre.Console
+open Spectre.Console.Cli
+open SpectreCoff
+
+type StatusSettings()  =
+    inherit CommandSettings()
+
+type StatusExample() =
+    inherit Command<StatusSettings>()
+    interface ICommandLimiter<StatusSettings>
+
+    override _.Execute(_context, _settings) =
+        let thinkingProcess (context: StatusContext) =
+            task {
+                do! Task.Delay(5000)
+                context.Status <- "More thinking"
+                context.SpinnerStyle <- Style.Parse("green")
+                context.Spinner <- Spinner.Known.Balloon2
+                do! Task.Delay(5000)
+            }
+
+        (startWithSpinnerStyle "Thinking" Spinner.Known.Pong (Style.Parse("red")) thinkingProcess).Wait()
```

Comment:
Is the type expected here the same as the one returned by the `toSpectreStyle `function in `Styling.fs`? If so, we don't expose the style anywhere else, so I'd suggest acception a `look` here as well. And I'd call it `startWithCustomSpinner` instead.

Reply:
That is a good point, changed as requested. Thanks @EluciusFTW !